.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Platform:**
```

Comment:
Just as an idea: maybe it is better to generalize the platform information and replace it with a *Conditions* section where a reporter describes the necessary prerequisites for reproducing the bug, e.g. if a bug occurs on every browser, filling out these fields do not add any useful information, however if, for example, a bug occurs only on Safari, the information about the browser is very valuable.

Reply:
I can make those changes